modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/JavaClientCodegen.java

```diff
@@ -377,6 +380,7 @@ public String toParamName(String name) {
 
     @Override
     public String toModelName(String name) {
+        name = super.toModelName(name);
```

Comment:
I agree. I think it's better for generators to opt-in to the prefix/suffix logic than to reimplement the base method just to remove prefix/suffix.